src/components/RowLink/RowLink.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import {Link} from '@gravity-ui/uikit';
+
+import {b} from './RowLink.classname';
+
+import './RowLink.scss';
+
+export const interactiveElementLinkRowClassName = b('interactive-element');
+
+type RowLinkProps<T> = {
+    entity: T;
+    index: number;
+    getLinkProps: (e: T, i: number) => {href: string; target: string};
+};
+
+export const RowLink = <T,>({getLinkProps, entity, index}: RowLinkProps<T>) => {
+    const content = React.useMemo(() => {
+        const linkProps = getLinkProps(entity, index);
```

Comment:
It's better to make this component as wrapper around Link, with the ability to pass any props.
```
<RowLink
    href={...}
    target={...}
/>
```
So there is no need in passing `getLinkProps`, `entity`, `index`.